bundles/monolith.yml

```diff
@@ -0,0 +1,121 @@
+version: '3.7'
```

Comment:
let's consider another name for this? 
we use it not only for monolith install but for install with services as well. 
Probably `magent_core`,`base_containers`, `backoffice` so on

Reply:
backoffice?

Reply:
I vote to backoffice

Reply:
test message

Reply:
@RuslanKostiv1  test 2